internal/api/v1/applications.go

```diff
@@ -233,6 +290,54 @@ func (hc ApplicationsController) Update(w http.ResponseWriter, r *http.Request)
 
 	return nil
 }
+
+func (hc ApplicationsController) Running(w http.ResponseWriter, r *http.Request) APIErrors {
```

Comment:
Can you add a comment on what this endpoint is all about? The response is not interesting so it should be used only to wait until and application is in running state. Let's document that a bit in comments.

Reply:
Done. Commit 1c96529a8810d7ed6c73dac70ab1d8498f212db6